UnityProject/Assets/Scripts/Objects/Doors/ConstructibleDoor.cs

```diff
@@ -58,7 +58,7 @@ public bool WillInteract(HandApply interaction, NetworkSide side)
 			if (Validations.HasUsedComponent<AirlockPainter>(interaction))
 				return true;
 
-			if (weldModule.CanDoorStateChange() == false && boltsModule.CanDoorStateChange() && doorMasterController.HasPower == false)
+			if (weldModule ? weldModule.CanDoorStateChange() == false : true && boltsModule ? boltsModule.CanDoorStateChange() == false : true && doorMasterController.HasPower == false)
```

Comment:
This is hard to read. I recommend encapsulating the conditions in methods/property so the name explains what is being checked